src/language/sv.tsx

```diff
@@ -80,12 +82,98 @@ const sv = {
         AddEntryView: 'Lägg till aktivitet',
```

Comment:
Are these capitalized to match the internal names of the screens?

Reply:
Yes